### DISCUSSION TITLE
start container from rootfs

### DISCUSSION THREAD

**Discussion by ashraffouda** [containers/buildah#5348] [General]
can we import containers/images from rootfs directory?
like if I did for example 
`pacstrap -K /var/chroot base base-devel` which will create `/var/chroot` can I use this directory to create an image/container from it?

**Comment by flouthoc** [comment]
Hmm does it work if you do 

```Dockerfile
FROM scratch
COPY /root-on-host /
```

and then run `buildah build -t imagename .` ?

**Comment by flouthoc** [comment]
Also `podman` allows you to run container directly from `rootfs` see https://docs.podman.io/en/latest/markdown/podman-run.1.html#rootfs

**Comment by ashraffouda** [comment]
I did something like this which seems to be working 
```
newcontainer=$(buildah from scratch)
scratchmnt=$(buildah mount $newcontainer)
pacstrap -c $scratchmnt base bash coreutils redis-server unzip
```

**Comment by rhatdan** [comment]
Yes this is what buildah was designed to do.
